samples/CosmosDBIntegration/CosmosDbIntegrationTutorial.md

```diff
@@ -0,0 +1,117 @@
+# Tutorial: Setup Redis Triggers for Azure Function in Visual Studio
```

Comment:
This is not really a tutorial as we see it here. What is the objective of someone who is trying to use this tutorial. 
Frame it this way: #CustomerIntent: As a < type of user >, I want < what? > so that < why? > .
As a developer of a cloud application, I want to use Redis Azure Functions to transfer data from the cache to a non-sql database like CosmosDB

If that were your intent, then that "story" is missing from this tutorial.